crates/wash-lib/src/lib.rs

```diff
@@ -28,14 +28,21 @@ pub mod parser;
 #[cfg(feature = "start")]
 pub mod start;
 
+#[cfg(feature = "nats")]
 pub mod actor;
+#[cfg(feature = "nats")]
 pub mod capture;
 pub mod common;
+#[cfg(feature = "nats")]
 pub mod config;
+#[cfg(feature = "nats")]
 pub mod context;
+#[cfg(feature = "nats")]
```

Comment:
Same comment re: `nats` feature flag on drain, can we remove this and have it build still?

Reply:
same situation as `context`